src/main/java/io/github/hsyyid/polis/cmdexecutors/DeleteClaimExecutor.java

```diff
@@ -0,0 +1,67 @@
+package io.github.hsyyid.polis.cmdexecutors;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import org.spongepowered.api.command.CommandException;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.spec.CommandExecutor;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.format.TextColors;
+
+import com.flowpowered.math.vector.Vector3i;
+
+import io.github.hsyyid.polis.Polis;
+import io.github.hsyyid.polis.utils.ConfigManager;
+
+public class DeleteClaimExecutor implements CommandExecutor{
+
+	@Override
+	public CommandResult execute(CommandSource src, CommandContext args) throws CommandException 
+	{
+		if (src instanceof Player)
+		{
+			Player player = (Player)src;
+			Optional<Vector3i> optionalChunk = Polis.game.getServer().getChunkLayout().toChunk(player.getLocation().getBlockPosition());
+			if (optionalChunk.isPresent())
+			{
+				Vector3i chunk = optionalChunk.get();
+				UUID worldUUID = player.getWorld().getUniqueId();	
+				if (!ConfigManager.isClaimed(chunk, worldUUID).equals("false"))
+				{
+					if (ConfigManager.isClaimed(chunk, worldUUID).equals("SafeZone"))
+					{
+						ConfigManager.unclaim("SafeZone", worldUUID, chunk.getX(), chunk.getZ());
+					}
+					
+					if (ConfigManager.isClaimed(chunk, worldUUID).equals("WarZone"))
+					{
+						ConfigManager.unclaim("WarZone", worldUUID, chunk.getX(), chunk.getZ());
+					}
+					
```

Comment:
This is done wrong. It should just get the Polis that it is claimed by, and unclaim it like that. I'll add this myself into `/polis adminunclaim`